components/slack/slack.app.mjs

```diff
@@ -383,6 +383,18 @@ export default {
       label: "Email",
       description: "An email address belonging to a user in the workspace",
     },
+    metadata_event_type: {
+      type: "string",
+      label: "Metadata Event Type",
+      description: "The name of the metadata event",
+      optional: true,
+    },
+    metadata_event_payload: {
+      type: "string",
+      label: "Metadata Event Payload",
+      description: "The payload of the metadata event",
+      optional: true,
+    },
```

Comment:
Quick question how are you sending this property as an object if you defined it as an string and I didn't see any parsing on this string either? thanks

Reply:
I was trying to work out how to define it as an object and noticed the `blocks` prop is a string so decided to copy that.
But I think you are right. I'll have to check if the blocks are getting parsed.

What would you recommend?